src/FeedbackForm/FeedbackForm.test.js

```diff
@@ -0,0 +1,37 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import FeedbackForm from "./";
+import React from "react";
+import userEvent from "@testing-library/user-event";
+
+describe("MultistepForm", () => {
+  const onSubmit = jest.fn();
+
+  beforeEach(() => {
+    render(<FeedbackForm onSubmit={onSubmit} />);
+  });
+
+  it("onSubmit is called when all fields pass Validation", async () => {
+    const floatingButton = screen.getByRole("button", { name: /add/i });
+    userEvent.click(floatingButton);
+
+    const feedback = await screen.findByTestId("SentimentSatisfiedAltIcon");
+    userEvent.click(feedback);
+
+    const title = await screen.findByRole("textbox", { name: /title/i });
+    userEvent.type(title, "Car Maker");
+
+    const description = screen.getByRole("textbox", { name: /description/i });
+    userEvent.type(description, "Car Maker");
+    userEvent.click(screen.getByRole("button", { name: /submit/i }));
+
+    await waitFor(() => {
+      expect(onSubmit).toHaveBeenLastCalledWith({
+        description: "Car Maker",
+        title: "Car Maker",
```

Comment:
It would be better to use different text for the title and description so we can guarantee they are correct and not mixed up in the code. For example, imagine you made mistake in the code and title was being returned as description, this test wouldn't catch it.